CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
```

Comment:
Hey out of curiosity how was this changelog generated?

Reply:
@o-leolleo Change log was automatically generated, but I used this task/opportunity to play around with github revision control so the changelog probably consists of history I rewrote.. apologies I didn't consider this